app/views/shared/ubiquity/collections/_show_document_list_row.html.erb

```diff
@@ -0,0 +1,71 @@
+<%#
+ Copied from https://github.com/samvera/hyrax/blob/v2.0.2/app/views/hyrax/dashboard/collections/_show_document_list_row.html.erb
+    and
+    https://github.com/samvera/hyrax/blob/v2.0.2/app/views/hyrax/collections/_show_document_list_row.html.erb
+    Consolidated since there is just a minor change of showing edit button when rendered via dashboard
+%>
+
+<% id = document.id %>
+<tr id="document_<%= id %>">
+  <td>&nbsp;
+    <% if current_user and document.depositor != current_user.user_key %>
+      <i class="glyphicon glyphicon-share-alt"/>
+    <% end %>
+  </td>
+  <td>
+    <div class="media">
+      <% if  document.thumbnail_id %>
+        <%= link_to [main_app, document], class: "media-left" do %>
+          <%= render_thumbnail_tag document, { class: "hidden-xs file_listing_thumbnail" }, { suppress_link: true } %>
+        <% end %>
+      <% else %>
+        <%= link_to "", class: 'media-left', 'aria-hidden' => true do %>
+          <span class="media-left hidden-xs file_listing_thumbnail" style="height:55px;width:50px;padding-left: 60px;" ></span>
+        <% end %>
+      <% end %>
+      <div class="media-body">
+        <h4 class="media-heading">
+          <%= link_to document.title_or_label, [main_app, document], id: "src_copy_link#{id}", class: "#{'document-title' if document.title_or_label == document.label}" %>
+          <a href="#" class="small" title="Click for more details"><i id="expand_<%= id %>" class="glyphicon glyphicon-chevron-right"></i></a>
+        </h4>
+        <%= render_collection_links(document) %>
+      </div>
+    </div>
+  </td>
+  <td>
+    <div class="media-body">
+      <%# UbiquityPress replaces document.creator.to_a.to_a pulled from document detail" %>
+      <span class="text-center"><%= display_json_values(document.creator).to_sentence %> </span>
+    </div>
+  </td>
+  <td>
+    <div class="media-body">
+      <span class="text-center"><%= document.date_uploaded %></span>
+    </div>
+  </td>
+  <td class="text-center">
+    <%= render_visibility_link(document) %>
+  </td>
+  <% if (controller.class == Hyrax::Dashboard::CollectionsController) && current_user %>
+    <td class="text-center">
+      <%= render 'show_document_list_menu', document: document %>
```

Comment:
I saw you removed "Action" from the headers in `_show_document_list` --> do we want to hide the actions menu here?

Reply:
The comments on that file explained that it consolidated since there is just a minor change of showing edit button when rendered via dashboard. So if log into your local hyku or the one on test, you will see that I am not hiding anything that wan't hidden before. When you access an individual collection through the dashboard the hidden actions are shown and when you do from outside it, the actions are hidden.
Try to look at the two links below pointing to the original files and check that section for both files and you will see that the original behaviour on Hyrax is preserved unchanged.

https://github.com/samvera/hyrax/blob/v2.0.2/app/views/hyrax/dashboard/collections/_show_document_list_row.html.erb
    
 https://github.com/samvera/hyrax/blob/v2.0.2/app/views/hyrax/collections/_show_document_list_row.html.erb

Reply:
I'm not sure to understand what you are saying. I was pointing out that since we are removing the "Action" header, maybe the intention was to remove the actions column / options altogether? See snapshots:

This is before any change from this PR
![screenshot from 2018-10-03 16-24-18](https://user-images.githubusercontent.com/26539307/46421171-7bd5dd00-c729-11e8-8535-c71bddcd2c61.png)

This is with changes from this PR - notice "Action" is removed but the drop-down remains - Is this the indended behaviour?
![screenshot from 2018-10-03 16-27-18](https://user-images.githubusercontent.com/26539307/46421211-914b0700-c729-11e8-8bd8-7012b6f55412.png)

Reply:
See the updated pull request. Thanks for the feedback.

Reply:
Good to go